operator/api/v1alpha1/common/common.go

```diff
@@ -45,3 +49,22 @@ type CheckType string
 
 const PreDeploymentCheckType CheckType = "pre"
 const PostDeploymentCheckType CheckType = "post"
+
+type KeptnMeters struct {
+	TaskCount          syncint64.Counter
+	TaskDuration       syncfloat64.Histogram
+	TaskActive         syncint64.UpDownCounter
+	DeploymentCount    syncint64.Counter
+	DeploymentDuration syncfloat64.Histogram
+	DeploymentActive   syncint64.UpDownCounter
+}
+
+const (
+	ApplicationName attribute.Key = attribute.Key("keptn.app_name")
+	Workload        attribute.Key = attribute.Key("keptn.workload")
+	Version         attribute.Key = attribute.Key("keptn.version")
+	Namespace       attribute.Key = attribute.Key("keptn.namespace")
```

Comment:
is this the target namespace where the App is being deployed?

Reply:
yes